sql/core/src/test/resources/sql-tests/results/postgreSQL/create_view.sql.out

```diff
@@ -887,35 +890,17 @@ AND NOT EXISTS (SELECT g FROM tbl4 LEFT JOIN tmptbl ON tbl4.h = tmptbl.j)
 -- !query 78 schema
 struct<>
 -- !query 78 output
-
+org.apache.spark.sql.AnalysisException
+Not allowed to create a permanent view `mytempview` by referencing a temporary view `tmptbl`;
```

Comment:
Ur, I see. This test should be passed (see: https://github.com/postgres/postgres/blob/43753c2cfbdd9870e68671abcc9dd377881071b0/src/test/regress/expected/create_view.out#L229) because views can depend on temp tables in pgSQL. So, can you replace `CREATE TEMP VIEW tmptbl` with `CREATE TABLE tmptbl`:
https://github.com/apache/spark/blob/be022d9aeeb6b39f7b51427d848600e15a100c30/sql/core/src/test/resources/sql-tests/inputs/postgreSQL/create_view.sql#L252

Reply:
@maropu I couldn't just change it to `CREATE TABLE` since it complains about `Hive support is required to CREATE Hive TABLE (AS SELECT);`. So, I created a file based table and introduced `INSERT INTO`. This required me to regenerate the golden file. Please let me know if this approach is OK with you. Thanks!

Reply:
Yea, I meant a data source table. That's ok.